crypto/keyring_streaming_test.go

```diff
@@ -0,0 +1,228 @@
+package crypto
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/pkg/errors"
+)
+
+const testFilename = "filename.txt"
+
+func TestKeyRing_EncryptStream(t *testing.T) {
+	keyRingPrivate, err := keyRingTestPrivate.Copy()
```

Comment:
Why copying? (Same for the public ones)

Reply:
I don't know I saw you did this in other tests so I just did the same. Should I remove it ?

Reply:
I think it copies when we somehow alter it, if not we can remove it :)